tools/stimulus/types-RR.yaml

```diff
@@ -258,11 +258,11 @@ EDGE_COLOR:
           }
         }
         if (!is.null(%I%)) {
-          %I% <- as.integer(%I%)-1L
+          %I% <- as.numeric(%I%)-1
```

Comment:
This looks like a bug fix?

Reply:
Yes it is. `R_SEXP_to_vector_int_copy()` should expect a real, not an integer.

Reply:
Here's a test for this:

```
count_isomorphisms(make_ring(3), make_ring(3), edge.color1=c(2,2,2), edge.color2=c(2,2,2), vertex.color1=c(3,3,3), vertex.color2=c(3,3,3), method='vf2')
```

This crashed before this fix. Could you please help me by adding this to the test suite?

Reply:
Never mind, it was easy enough. See #1050